### ISSUE TITLE
Build Customer View (Jade)

### ISSUE THREAD

**Issue by luchia** [TeamMentor/TM_4_0_Design#721]

**Comment by luchia** [comment]
This has been done and tested in the three main Mac browsers. @roman87 @DinisCruz, I'm not sure how you want to implement this but the feature is there for when we are ready. This is currently only on the Lucy branch -

![screen shot 2015-04-22 at 10 09 22 am](https://cloud.githubusercontent.com/assets/4547737/7265464/c8837ba4-e8dd-11e4-93f8-d82ef7ee3dab.png)
![screen shot 2015-04-22 at 10 09 25 am](https://cloud.githubusercontent.com/assets/4547737/7265465/c892636c-e8dd-11e4-9632-a75ccac62c59.png)
![screen shot 2015-04-22 at 10 09 34 am](https://cloud.githubusercontent.com/assets/4547737/7265466/c8958056-e8dd-11e4-9bba-74929a5712e2.png)
![screen shot 2015-04-22 at 10 09 37 am](https://cloud.githubusercontent.com/assets/4547737/7265467/c896c61e-e8dd-11e4-808a-db7bccc9f3f4.png)
![screen shot 2015-04-22 at 10 09 41 am](https://cloud.githubusercontent.com/assets/4547737/7265468/c89bef9a-e8dd-11e4-9285-812b5e6ee903.png)
![screen shot 2015-04-22 at 10 09 47 am](https://cloud.githubusercontent.com/assets/4547737/7265469/c89ce846-e8dd-11e4-895a-728a4174a91c.png)
![screen shot 2015-04-22 at 10 09 50 am](https://cloud.githubusercontent.com/assets/4547737/7265470/c8be1ffc-e8dd-11e4-9ec2-c241bb74fdd4.png)
![screen shot 2015-04-22 at 10 09 54 am](https://cloud.githubusercontent.com/assets/4547737/7265471/c8c884a6-e8dd-11e4-84b5-8ec5f4bc286f.png)
![screen shot 2015-04-22 at 10 09 58 am](https://cloud.githubusercontent.com/assets/4547737/7265472/c8da49c0-e8dd-11e4-91cb-7aea53d1ba15.png)

**Comment by luchia** [comment]
Assigning to @roman87 so I know what to do with it next.

**Comment by DinisCruz** [comment]
I dont get this issue, what am I looking for?

also why is there two logos on the top banner?

**Comment by luchia** [comment]
Roman asked me to build a template for the client where they have their own navigation bar in Jade. The tm logo is a placeholder in place of the client logo.

**Comment by DinisCruz** [comment]
ahh ok, I get it now, thx

**Comment by roman87** [comment]
Ok, can we get the logos from lloyds teammentor.net mckesson.teammentor.net and sony.teammentor.net and see how they look. This also needs to be somhow configurable so we can easily change the top banner in the future.

**Comment by luchia** [comment]
So I had that in mind when I made this because I knew we'd need to change colours etc in the customer navigation. It has it's own div (#customer-navigation) which currently has no styling so will be easy enough for customisation to be added.

![screen shot 2015-05-05 at 5 46 41 am](https://cloud.githubusercontent.com/assets/4547737/7461269/5b43dfb6-f2ec-11e4-9c2b-2d28f6537599.png)
![screen shot 2015-05-05 at 5 50 43 am](https://cloud.githubusercontent.com/assets/4547737/7461272/5b45aabc-f2ec-11e4-864d-bd557cd9b6c1.png)
![screen shot 2015-05-05 at 5 58 14 am](https://cloud.githubusercontent.com/assets/4547737/7461273/5b47a4e8-f2ec-11e4-9846-24e2fc7cfe4a.png)
![screen shot 2015-05-05 at 5 58 50 am](https://cloud.githubusercontent.com/assets/4547737/7461274/5b48f56e-f2ec-11e4-9711-017849bd1a9c.png)
![screen shot 2015-05-05 at 6 02 00 am](https://cloud.githubusercontent.com/assets/4547737/7461278/5b713308-f2ec-11e4-9e5f-72d0fb236e23.png)
![screen shot 2015-05-05 at 6 02 04 am](https://cloud.githubusercontent.com/assets/4547737/7461277/5b707986-f2ec-11e4-8e11-f0ceaa2d8a0f.png)

I think we should add a requirement that the logo submitted must be a certain size so that we can actually have an easy time with these customizations. It's too hard trying to build something for every single size but we have constraints, then we can automate it - which would be best.

**Comment by roman87** [comment]
Ok awesome. So @DinisCruz now we need a way to manage these customizations similar to what we do with Site_Data. It would be good if we could leverage the same Site_Data repos. Maybe add TM40 directory there and customizations there. We should then also add a config file option in TM40 to enable/disable customization and the customization directory.

**Comment by luchia** [comment]
Assigning to Dinis.

**Comment by DinisCruz** [comment]
This is now done with a number of options to customise it.

The more advanced example can be seen at the master branch of https://github.com/TMClients/SiteData_TM

There is a simple example on the tp-example branch https://github.com/TMClients/SiteData_TM/tree/tp-example which shows how to add a logo to the top of the page.

![image](https://cloud.githubusercontent.com/assets/656739/8131718/19db8394-1114-11e5-8d4c-86b7b7e1db2a.png)

When that value is set, it will be picked up by 

![image](https://cloud.githubusercontent.com/assets/656739/8131725/2aed8434-1114-11e5-8b7b-3857b9226411.png)

and shown at

![image](https://cloud.githubusercontent.com/assets/656739/8131736/40b4de3e-1114-11e5-849e-e3736d1ef388.png)

You can see these in action if you switch between the branches (node server restart is needed)

![image](https://cloud.githubusercontent.com/assets/656739/8131749/53354512-1114-11e5-9c87-d1dc1004d131.png)

**Comment by DinisCruz** [comment]
@michaelhidalgo and @luchia can you give this a test drive

I expect a couple more jade files will need to be changed (@luchia can you look into it)

**Comment by michaelhidalgo** [comment]
I've been working on the customization for Sony, we will need some design changes, but it is easy to add their content, this is the first version (POC) for the Portal

![screen shot 2015-06-12 at 12 49 08](https://cloud.githubusercontent.com/assets/1732035/8137723/6ee01916-1102-11e5-9e38-0faf69a9618e.png)

**Comment by luchia** [comment]
@michaelhidalgo, I pushed to a new branch (Issue_721) with the files you requested. 

You have the following files to use now -
![screen shot 2015-06-16 at 9 51 01 am](https://cloud.githubusercontent.com/assets/4547737/8173212/76da1028-140d-11e5-8fb3-fb09b461c219.png)

Where the logged out one is for the portal page and the logged in one is for the application view. I have also created a mixin file in the SiteData/TM area that has the sony login mixin. I put it there because I wasn't sure where you'd like it.

I've also added a new .team-professor button class that will make that link look as it should. If you apply the .button class to both of those links (and then the .team-professor class to the TP link), then the links will look like buttons and be the right colours. I've also added the Sony tile to the assets/clients directory. This can all be found on the Issue_721 branch in the TM_Static repo.
